src/server/scripts/Events/hallows_end.cpp

```diff
@@ -1304,6 +1326,8 @@ struct boss_headless_horseman_head : public ScriptedAI
             me->CastSpell(me, SPELL_THROW_HEAD_BACK, true);
             if (Unit* owner = GetOwner())
                 owner->RemoveAura(SPELL_HORSEMAN_IMMUNITY);
+            
```

Comment:
Workflow claims there is a white space here.

Reply:
You're right, there was.